Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{etc}/fpc.cfg"
   end
 
   test do
     hello = <<-EOS.undent
       program Hello;
+      uses GL;
```

Comment:
I don't think so, the whole point of this additional step is that fpc will read #{etc}/fpc.cfg and won't need any additional path

in fact it works for me locally (the output i posted is from my shell), and I'm quite sure I saw a green merged button a few revisions ago.

Maybe the use of #{etc} is not appropriate in this case, and instead it should use #{prefix}/etc?